### ISSUE TITLE
[BUG] Unable to start extension 

### ISSUE THREAD

**Issue by mattywarr** [microsoft/vscode-dotnettools#1256]
### Describe the Issue

When trying to use a version higher than 1.4.29, the extension fails to start on my computer and I don't know why. Only started happening late last week when I updated to latest prerelease 1.8.10 from 1.8.8. Rolling back to any version since to 1.4.29 does not work.

**C# Extension Output Log**
```
[Error - 9:19:31 AM] Microsoft.CodeAnalysis.LanguageServer client: couldn't create connection to server.
SyntaxError: Unexpected token '... is not valid JSON
	at JSON.parse (<anonymous>)
	at f (c:\Users\Matthew.Warr\.vscode\extensions\ms-dotnettools.csdevkit-1.8.10-win32-x64\dist\extension.js:2:1426415)

```
![image](https://github.com/microsoft/vscode-dotnettools/assets/19187916/b06f211b-c735-4d09-8dcc-543c910e8e86)

**C# Dev Kit Output Log**
```
Saving VSCode common properties failed: SyntaxError: Unexpected token '...... is not valid JSON
#getSession: Failed to running method. Error: 'Canceled: Canceled'
Starting Spawn .NET server...
#getSession: Failed to running method. Error: 'Canceled: Canceled'
#getSession: Failed to running method. Error: 'Canceled: Canceled'
More than one solution files in the workspace. Waiting for the developer to make a choice.
Using preinstalled .NET runtime at "C:\Program Files\dotnet\dotnet.exe"
.NET server STDERR: Unhandled exception. 
.NET server STDERR: System.ArgumentNullException: Value cannot be null. (Parameter 'filePath')
   at Microsoft.VisualStudio.Telemetry.CommonProperty.CommonPropertyParserFactory.GetCommonPropertyParser(TelemetrySessionInternalBase session)
   at Microsoft.VisualStudio.Telemetry.TelemetrySessionInternalBase.LoadCommonProperties()
   at Microsoft.VisualStudio.ManagedTelemetry.VSCodeTelemetry.CreateAndStartDefaultSession(String sessionSettings)
   at Microsoft.VisualStudio.ManagedTelemetry.VSCodeTelemetry.StartTelemetrySessionAndCollector(String vsCodeTelemetryLevel, String customSessionId)
   at Microsoft.VisualStudio.Server.ServiceBroker.Program.<>c.<<BuildCommandLine>b__7_1>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Invocation.AnonymousCommandHandler.InvokeAsync(InvocationContext context)
   at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass18_0.<<UseParseErrorReporting>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass13_0.<<UseHelp>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass20_0.<<UseTypoCorrections>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__19_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseDirective>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__6_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<CancelOnProcessTermination>b__1_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Invocation.InvocationPipeline.InvokeAsync(IConsole console)
   at System.CommandLine.Parsing.ParseResultExtensions.InvokeAsync(ParseResult parseResult, IConsole console)
   at System.CommandLine.Parsing.ParserExtensions.InvokeAsync(Parser parser, String[] args, IConsole console)
   at Microsoft.VisualStudio.Server.ServiceBroker.Program.Main(String[] args)
   at Microsoft.VisualStudio.Server.ServiceBroker.Program.<Main>(String[] args)

Error occurred in launchDotNetServer: Error: Stream terminated before required bytes were read.
Failed at 'Spawn .NET server' (7213ms) with error: Error: Stream terminated before required bytes were read.
.NET server exited with 3762504530

```
![image](https://github.com/microsoft/vscode-dotnettools/assets/19187916/7017ec9e-8ec9-419c-8ee4-d7a62afa11c8)

### Steps To Reproduce

Install the extension higher than 1.4.29 on my computer. 
It appears to be environmental in some way so not really reproducible (Works OK on my Mac)

### Expected Behavior

I expect the extension to start

### Environment Information

Windows 10
VS Code 1.90.2 System (Though user version is the same)

**Comment by dositec** [comment]
Ou, I have same, I thought it's only my local bug or it's sanction bug because I'm from Russia. I have installed VSCodium but out of the box it cannot debug tests.

**Comment by dibarbet** [comment]
I don't believe this is Roslyn.  Roslyn is failing to start because DevKit is failing to start.  Looks like an issue reading telemetry properties?

Tagging @jonathanjyi

**Comment by jonathanjyi** [comment]
@mattywarr could you provide a copy of the file located at:
%Temp%\csdevkit\Telemetry\Resources.json

Please <REDACT> any personal information you need to.

**Comment by jonathanjyi** [comment]
@mattywarr , @dositec if you delete the files under %Temp%\csdevkit and restart, does it resolve your issue?

**Comment by mattywarr** [comment]
No need to redact, the resources file looks borked.
![image](https://github.com/user-attachments/assets/0d23ef09-ce49-4ac2-bf7b-103d8c66a77c)

But as you indicated, deleting the contents of the temp folder seems to have resolved it!

**Comment by dositec** [comment]
I 

> No need to redact, the resources file looks borked. ![image](https://private-user-images.githubusercontent.com/19187916/349482381-0d23ef09-ce49-4ac2-bf7b-103d8c66a77c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyMjY5NDEsIm5iZiI6MTcyMTIyNjY0MSwicGF0aCI6Ii8xOTE4NzkxNi8zNDk0ODIzODEtMGQyM2VmMDktY2U0OS00YWMyLWJmN2ItMTAzZDhjNjZhNzdjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE3VDE0MzA0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1MjA0ZWRhZjZiNWRjOTI4YzI0MDI0N2MwY2U2MzM0Y2ZjNzI5ZTFiMjcxZjljZDFmMzAxMDJiYmUzYmIxMDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oYFDAFkmPKusNmy5wb6AInuMlOuLuFw-UhEP6LwAlOc)
> 
> But as you indicated, deleting the contents of the temp folder seems to have resolved it!

I had the same file content but I have fixed the problem, I cleanup temp folder, and tuned bios to normal mode(was fast) + disable "x.m.p." (motherboard - ASUS).

**Comment by jonathanjyi** [comment]
Thanks for the updates @dositec @mattywarr , I'm investigating how these files are getting corrupted now.

**Comment by webreidi** [comment]
Fixed in v1.9.55. Reopen if still experiencing the problem.
